test/plugins/FakeTimeStampDataGenerator.hpp

```diff
@@ -0,0 +1,88 @@
+/**
+ * This is part of the DUNE DAQ Software Suite, copyright 2020.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#ifndef TRIGGER_TEST_PLUGINS_FAKETIMESTAMPDATAGENERATOR_HPP_
+#define TRIGGER_TEST_PLUGINS_FAKETIMESTAMPDATAGENERATOR_HPP_
+
+#include "dune-trigger-algs/TimeStampedData.hh"
+
+#include "appfwk/DAQModule.hpp"
+#include "appfwk/DAQSink.hpp"
+#include "appfwk/ThreadHelper.hpp"
+
+#include <ers/Issue.hpp>
+
+#include <memory>
+#include <string>
+#include <vector>
+#include <chrono>
+#include <random>
+
+namespace dunedaq {
+namespace trigger {
+
+/**
+ * @brief RandomDataListGenerator creates vectors of ints and writes
+ * them to the configured output queues.
+ */
+class FakeTimeStampDataGenerator : public dunedaq::appfwk::DAQModule
+{
+public:
+  /**
+   * @brief FakeTimeStampDataGenerator Constructor
+   * @param name Instance name for this FakeTimeStampDataGenerator instance
+   */
+  explicit FakeTimeStampDataGenerator(const std::string& name);
+
+  FakeTimeStampDataGenerator(const FakeTimeStampDataGenerator&) =
+    delete; ///< FakeTimeStampDataGenerator is not copy-constructible
+  FakeTimeStampDataGenerator& operator=(const FakeTimeStampDataGenerator&) =
+    delete; ///< FakeTimeStampDataGenerator is not copy-assignable
+  FakeTimeStampDataGenerator(FakeTimeStampDataGenerator&&) =
+    delete; ///< FakeTimeStampDataGenerator is not move-constructible
+  FakeTimeStampDataGenerator& operator=(FakeTimeStampDataGenerator&&) =
+    delete; ///< FakeTimeStampDataGenerator is not move-assignable
+
+  void init(const nlohmann::json& obj) override;
+  void get_info(opmonlib::InfoCollector& ci, int level) override;
+
+private:
+  // Commands
+  void do_configure(const nlohmann::json& obj);
+  void do_start(const nlohmann::json& obj);
+  void do_stop(const nlohmann::json& obj);
+  void do_scrap(const nlohmann::json& obj);
+
+  // Threading
+  dunedaq::appfwk::ThreadHelper thread_;
```

Comment:
The DAQ style guide naming convention for member variables is `m_variable_name`, so this should become `m_thread`. The same applies to the other member variables.